docs/database_migration_test.md

```diff
@@ -0,0 +1,47 @@
+## Testing database migration
+
```

Comment:
Thanks for the doc!

Not a big deal, but for consistency, it could be great to have a table of content.

Just manually add these 2 lines here:

```
<!--- TOC -->

<!--- END -->
```
 and run `./gradlew knit` to generate it.